### ISSUE TITLE
Add content for topic: Basic HTTP

### ISSUE THREAD

**Issue by DanielOrmeno** [N0obCoding/Introduction-to-the-web#4]
# Add course content for Basic HTTP.

## Overview
This module is intended to cover the basics of HTTP. It's intended to cover the different types of requests, status codes and an overview of the anatomy of an HTTP request and response. What differentiates HTTP from HTTPS and why do we care about HTTPS. 

## Guidelines
**Partial contributions are welcome!** You are not expected to cover or provide all the content for this topic, all types of contributions are encouraged!

You are encouraged to take credit for your contribution, drop your name in the **list of contributors** for each topic (found in the topic's `readme.md` file)

All content must be added to [the http folder](https://github.com/N0obCoding/Introduction-to-the-web/tree/master/http).

When submitting your PR, if possible, provide a rough estimate of how long it will take to consume the content.

Please update the links in the table of contents in the [main README.md](https://github.com/N0obCoding/Introduction-to-the-web/blob/master/README.MD) file as part of your PR.

### File structure
When creating course content, please use the following file structure.

```
 Introduction-to-the-web
 |-- assets
 |    |--> Module's common images and files
 |-- http
 |    |-- readme.md -> topic content
 |    |-- assets
 |    |     |--> Topic specific assets 
 |    |-- exercises
 |    |     |--> Topic specific exercises (if any) 
```

### On contributing content and Plagiarism
Please be mindful with the content you include. If you quote or use content created by someone else, always give credit to the original author and make sure that you have their permission to share their content.

Contributions that copy commercial courses, or that fail to include references to the original source will be removed or declined.

**Comment by RichardLim00** [comment]
Hello, I would like to help out on this issue. But I'm quite a rookie myself in web-dev (spent around 10 months learning.), is it okay?

**Comment by DanielOrmeno** [comment]
By all means! All contributions are welcome. We'll polish the content as we go. Thank you for the interest!

**Comment by godcrampy** [comment]
I'd like to contribute and expand my _http_ knowledge! I had two queries:
1. What should the level of difficulty explanation for the content? Should I write ELI5 type explanations or can I assume some familiarity of the audience with computers in general.
2. Just to be sure, you want all the explanation markdown files as well as images in the assets directory right?

Regards 😄

**Comment by RichardLim00** [comment]
I have made a pull request for HTTP, but I left out HTTPS as I have yet to study it. I think it should be done by godcrampy, he looks more knowledgable.

**Comment by DanielOrmeno** [comment]
Hi @godcrampy yes! the content should be included in the markdown files and any additional assets should live in the assets folder.

With regards to the level of difficulty, this is a beginner (Starting from zero) course so the narrative of the content should be targeting non-tech people, ELI5 should work fine! The idea is that we can all make the content grow, so any further contributions to the content that is already there are encouraged!

**Comment by DanielOrmeno** [comment]
I'm still looking for contributors to enhance the current content.
